webpack/assets/javascripts/react_app/components/CPUCoresInput/cpuCoresInput.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../common/colors';
+
+.form-group {
+  display: flex;
+}
+
+.warning-icon {
```

Comment:
Too generic, might affect other components who want to use the `warning-icon` class.